code/modules/clothing/spacesuits/rig/suits/alien.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/weapon/rig/unathi
+	name = "NT breacher chassis control module"
+	desc = "A cheap NT knock-off of an Unathi battle-rig. Looks like a fish, moves like a fish, steers like a cow."
+	suit_type = "NT breacher"
+	icon_state = "breacher_rig_cheap"
+	armor = list(melee = 60, bullet = 60, laser = 60, energy = 60, bomb = 70, bio = 100, rad = 50)
+	emp_protection = -20
+	slowdown = 6
+	offline_slowdown = 10
+	vision_restriction = 1
+	offline_vision_restriction = 2
+	
+	chest_type = /obj/item/clothing/suit/space/new_rig
+	helm_type = /obj/item/clothing/head/helmet/space/new_rig/unathi
+	boot_type = /obj/item/clothing/shoes/magboots/rig/unathi
+
+/obj/item/weapon/rig/unathi/fancy
+	name = "breacher chassis control module"
+	desc = "An authentic Unathi breacher chassis. Huge, bulky and absurdly heavy. It must be like wearing a tank."
+	suit_type = "breacher chassis"
+	icon_state = "breacher_rig"
+	armor = list(melee = 90, bullet = 90, laser = 90, energy = 90, bomb = 90, bio = 100, rad = 80) //Takes TEN TIMES as much damage to stop someone in a breacher. In exchange, it's slow.
```

Comment:
Not asking you to address this, but seriously Bay? what made you guys think that giving someone 90% damage reduction (and melee immunity with a security jumpsuit) was a remotely sane idea?